Resources/Locale/en-US/interaction/interaction-popup-component.ftl

```diff
@@ -27,6 +27,7 @@ petting-failure-crab = You reach out to pet {THE($target)}, but {SUBJECT($target
 petting-failure-goat = You reach out to pet {THE($target)}, but {SUBJECT($target)} stubbornly refuses!
 petting-failure-goose = You reach out to pet {THE($target)}, but {SUBJECT($target)} {CONJUGATE-BE($target)} too horrible!
 petting-failure-sloth = You reach out to pet {THE($target)}, but {SUBJECT($target)} somehow dodge with ludicrous speed!
+petting-failure-holo = You reach out to pet {THE($target)}, but it's spikes almost impale your hand!
 
```

Comment:
Change "it's" to {POSS-ADJ($target)} and you should be good. Also maybe close this PR and make this change in your first branch

Reply:
>Also maybe close this PR and make this change in your first branch

Ignore this part now.